### PR TITLE
Fixes #1846

### LINKED ISSUE #1846: Create UI for a "detect bugs with similar descriptions" feature
It's from https://github.com/webcompat/webcompat.com/issues/1826#issuecomment-333793412

**Intro:**
When there is Firefox regression bugs, we'll see users report same issues but they have same root cause. It comes into my mind if we can let user know other users already reported similar issue, they can see what's happening and vote/+1/vouch same issue happen to him/her.

**Description:**
While user is typing in the description and more than 2 words, use the keyword to search existing issue description and find similar description.

Click on the suggested issue opens a new tab, see what's going on and do the following. Agree with +1 or disagree then close the tab.

**TBD:**
- What's the better way to guide users have discussion similar description in an issue?
- It's doable via github search API or we need another DB/end point to do this?

### PR DESCRIPTION
**Pull request by MDTsai**
First prototype to make sure this is what we want for similar bug suggestion. r?=@zoepage

### CODE REVIEW

File: webcompat/static/js/lib/bugform.js [hunk 1]
```diff
@@ -216,6 +219,60 @@ function BugForm() {
...
+    }
+
+    if (this.suggestDelay) {
+      clearTimeout(this.suggestDelay);
+    }
+    var bugform = this;
+    this.suggestDelay = setTimeout(function() {
+      try {
+        var url = new URL(bugform.urlField.val());
+        if (url.hostname.length > 7) {
```
Comment (zoepage) [inline webcompat/static/js/lib/bugform.js]: https://github.com/webcompat/webcompat.com/issues/1826#issuecomment-332374279 > "About the search API" 

> "... fire the event starting at the 5th character ..."

Would be great, if we could change the 7 to a 4 then. thanks!

**Comment by zoepage** [comment]
@MDTsai one small change. otherwise the code looks good. :)

File: webcompat/static/js/lib/bugform.js [hunk 2]
```diff
@@ -216,6 +219,60 @@ function BugForm() {
...
+          var query = `https://api.github.com/search/issues?page=1&per_page=10&stage=all&sort=created&q=${url.hostname}+in:title+repo%3Awebcompat%2Fweb-bugs&order=desc`;
+
+          fetch(query)
```
Comment (miketaylr) [inline webcompat/static/js/lib/bugform.js]: This is fine for a prototype, but we need to be maximally compatible with old browsers for bug reporting:

https://caniuse.com/#feat=fetch

Comment (MDTsai) [reply webcompat/static/js/lib/bugform.js]: +1, so this is only for prototyping, let's discuss this in all hands to know if this really improve bug quality? Any change could make it better?

Comment (zoepage) [reply webcompat/static/js/lib/bugform.js]: +1

File: webcompat/static/js/lib/bugform.js (hunk 1 above)
Comment (zoepage) [reply webcompat/static/js/lib/bugform.js]: @MDTsai Would you like to make the small fix? Or shall I add a follow up? I don't want to block the merge with this small request. :)

Comment (zoepage) [reply webcompat/static/js/lib/bugform.js]: let's discuss the char length at some other point. Merging, so team can review.

**Comment by miketaylr** [comment]
@zoepage @MDTsai FYI

https://travis-ci.org/webcompat/webcompat.com/builds/315353544#L612

**Comment by zoepage** [comment]
That happens due the ES6 features as `grunt-contrib-uglify` v2 is all about ES5. On it!

**Pull request merged**